src/views/design/ScriptDetail/EditAction/index.tsx

```diff
@@ -168,6 +177,10 @@ function EditAction({ onClose, action, onEdit, state }: IPublicProps & IObserveP
                             className={classes.conditionTextField}
                             defaultValue={condition}
                             onBlur={(e) => setCondition(e.target.value)}
+                            InputProps={{
+                                readOnly: !checkAuthority(SECURITY_PRIVILEGES.S_SCRIPTS_WRITE, 'PUBLIC'),
```

Comment:
Can we already use the security group name  of the script in question instead of the fixed value 'PUBLIC'